querydsl-collections/src/test/java/com/querydsl/collections/LeftJoinTest.java

```diff
@@ -62,7 +62,7 @@ public void List() {
     }
 
     @Test
-    public void Alias() {
+    public void alias1() {
```

Comment:
To me the `_` made sense, since it is the smallest, non-intrusive, way of escaping.
if you use any number I instantly look for the other `2`, `3` etc.. cases.
@johnktims Your opinion?